Bottom: 4fb1fac11560c35ea3996c171e030875a3a2a84f
Top:    29fba5d792e6d7f5fe52fba37da323c8d12eade5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-03 22:57:31 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 59e5e31..6089eba 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
